vm/src/frame.rs

```diff
@@ -85,7 +85,7 @@ pub struct Frame {
     stack: RefCell<Vec<PyObjectRef>>, // The main data frame of the stack machine
     blocks: RefCell<Vec<Block>>,      // Block frames, for controlling loops and exceptions
     pub scope: Scope,                 // Variables
-    pub lasti: RefCell<usize>,        // index of last instruction ran
+    lasti: RefCell<usize>,            // index of last instruction ran
```

Comment:
If you're going to touch this, do you want to make it a `Cell<usize>`? It really doesn't need to be a `RefCell`

Reply:
Agreed!